nvflare/app_common/widgets/log_receiver.py

```diff
@@ -0,0 +1,71 @@
+# Copyright (c) 2021-2022, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import os
+from logging import LogRecord
+from typing import List, Optional
+
+from nvflare.apis.analytix import AnalyticsData, AnalyticsDataType
+from nvflare.apis.dxo import from_shareable
+from nvflare.apis.fl_constant import LogMessageTag
+from nvflare.apis.fl_context import FLContext
+from nvflare.apis.shareable import Shareable
+from nvflare.app_common.widgets.streaming import AnalyticsReceiver
+
+
+class LogAnalyticsReceiver(AnalyticsReceiver, logging.StreamHandler):
+    CLIENT_LOG_FOLDER = "client_log"
+
+    def __init__(self, events: Optional[List[str]] = None, formatter=None):
+        super().__init__(events=events)
```

Comment:
If provided "events" is None, we should set them to [_LOGGING_EVENT_TYPE,  "fed."+_LOGGING_EVENT_TYPE]. Otherwise the super class would set them to the event types for analytic data events.
_LOGGING_EVENT_TYPE is defined in streaming.py.

Reply:
Done